chord_metadata_service/chord/workflows/wdls/cbioportal_export.wdl

```diff
@@ -45,15 +41,14 @@ task dataset_id_from_project_dataset {
         python3 -c 'print("~{project_dataset}".split(":")[1], end="")'
     >>>
     output {
-        String dataset_id = stdout()
+        String dataset_id = read_string(stdout())
```

Comment:
Why is this necessary?

Reply:
it didn't work before; `stdout()` resolves to the path of stdout, not the actual stdout content